spacy/cli/project/remote_storage.py

```diff
@@ -123,12 +125,24 @@ def find(
         elif command_hash is not None:
             urls = list((self.url / name / command_hash).iterdir())
```

Comment:
If you use a local path and get here but the directory doesn't exist, there's an uncaught exception. This can happen if you do this:

1. Run some but not all steps with tracked outputs
2. Push
3. Pull

I think it could also happen if you pull before pushing.

```suggestion
            try:
                urls = list((self.url / name / command_hash).iterdir())
            except FileNotFoundError:
                urls = []
```